README.md

```diff
@@ -1,14 +1,65 @@
-Data Shapes WG Repository
-=====
+![](w3c-no-bars.svg)
+
+# Data Shapes WG Repository
+
+This repository is the working space of the [W3C's Data Shapes Working Group](https://www.w3.org/groups/wg/data-shapes/). 
+
+The WG is currently working on updated and new [SHACL](https://www.w3.org/TR/shacl/) recommendations, as per the [WG's Charter](https://www.w3.org/2024/12/data-shapes.html).
+
+## Deliverables:
+
+#### Phase 1, prioritised until July 2025.
```

Comment:
July isn't noted in the charter.  Is there a reason that month is mentioned here?  I'm particularly thinking of the RDF 1.2 dependencies.

Reply:
It was just a timeline we set ourselves - 1st half of 2025 - but yes, the real yardstick is RDF 1.2 WG